tests/foreman/api/test_repository.py

```diff
@@ -1606,7 +1602,7 @@ def test_negative_synchronize_private_registry_no_passwd(
         with pytest.raises(
             HTTPError,
             match='422 Client Error: Unprocessable Entity for url: '
-            f'{default_sat.url}/katello/api/v2/repositories',
+            f'{default_sat.url}:443/katello/api/v2/repositories',
```

Comment:
Hi @sambible , do you really need to add `443` here? That is is the default port for HTTPS and when you run the test you can see the scheme is "https":

    >>> default_sat.url                                                                    
    'https://<redacted>.redhat.com'

Reply:
Ok, sorry, I see that is the error message you are checking, /o\